src/server/components/annual-review/controller.ts

```diff
@@ -120,13 +120,12 @@ export async function confirmPostController(req: Request, res: Response, next: N
       return res.redirect(`/annual-review/confirm/${req.body.reference}`);
     }
 
-    if (chosenValue === "yes") {
-      return res.redirect(`/annual-review/declaration/${req.body.reference}`);
-    }
-
+    let updatesRequired = false;
     if (chosenValue === "no") {
-      await redirectToFormRunner(req, res, next);
+      updatesRequired = true;
     }
```

Comment:
fine as it is, but can be simplified with just `const updatesRequired = chosenValue === "yes"`;

Reply:
done